pyproject.toml

```diff
@@ -32,38 +61,172 @@ ignore = """
 tests
 """
 
-[tool.pylint.messages_control]
-disable = """
-bad-continuation,line-too-long,too-few-public-methods,import-error,too-many-instance-attributes, too-many-arguments,
-raise-missing-from, invalid-name, too-many-lines
-"""
+[build-system]
+requires = ["poetry>=0.12"]
+build-backend = "poetry.masonry.api"
+
+[tool.isort]
+multi_line_output = 3
+include_trailing_comma = true
+force_grid_wrap = 0
+use_parentheses = true
+line_length = 120
+
+[tool.black]
+target-version = ["py310"]
+line-length = 120
+include = '\.pyi?$'
+exclude = '''
+(
+  /(
+      \.eggs         # exclude a few common directories in the
+    | \.git          # root of the project
+    | \.hg
+    | \.mypy_cache
+    | \.tox
+    | \.venv
+    | _build
+    | buck-out
+    | build
+    | dist
+  )/
+  | foo.py           # also separately exclude a file named foo.py in
+                     # the root of the project
+)
+'''
+
+[tool.pytest.ini_options]
+addopts = [
+    "--import-mode=importlib",
+]
+log_cli = true
+log_cli_level = "INFO"
 
 [tool.pylint.basic]
 good-names = """
 logger,i,j,k,ex,Run,_
 """
```

Comment:
pylint has been removed from the pre-commit file and has been replaced by `ruff`. So it should be removed from here as well. hould also be removed since it's not used any more. The dev dependency should also be removed.

Reply:
here

Reply:
done